.gitignore

```diff
@@ -96,3 +96,4 @@ src/TestResults
 test/core/customast/test.txt
 test/core/files/test.png
 test/core/migration/writetext.txt
+src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs
```

Comment:
@manuelsaldivar525 please revert this - this file should not be added to the git ignore - we need to commit it sometimes. Just do not include the auto generated changes in your commit.

Reply:
Yes, I already reverted it.